src/Sentry.Maui/Internal/MauiEventsBinder.cs

```diff
@@ -409,6 +416,40 @@ private void OnElementOnFocused(object? sender, FocusEventArgs _) =>
     private void OnElementOnUnfocused(object? sender, FocusEventArgs _) =>
         _hub.AddBreadcrumbForEvent(_options, sender, nameof(VisualElement.Unfocused), SystemType, RenderingCategory);
 
+    internal void OnElementLoaded(object? sender, EventArgs _)
```

Comment:
Consider using IMauiElementEventBinder which allows you to work this as more "plugin" meaning that you can optionally plugin additional logic and hooks.  This interface/mechanic still needs docs (and I think I may hate the breadcrumb callback - don't ask why I did... I don't have a logical answer anymore).

Reply:
@jamescrosswell It's a Sentry interface I added to Sentry.Maui so we didn't have to keep bolting stuff into that one super class.  It allows external users to technically "auto" breadcrumb/hook their own stuff

Reply:
Ahh, I see. Cool, I'll take a look!